New package: MosaikAPIv3 v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: MosaikAPIv3
- Repository: https://gitlab.com/mosaik/api/MosaikAPIv3.jl
- Created by: https://gitlab.com/stomberg.malte
- Version: v1.0.0
- Commit: d69fe59bbfbe1201a4f834919e9e08940d4ee438
- Git reference: HEAD

rafaqz commented:
You will need docs or at least a readme to register this.

goerz commented:
[noblock] It would still be nice to also add a paragraph in the README to introduce the package before linking out to the full documentation

JuliaHubRegistrator committed:
New package: MosaikAPIv3 v1.0.0

UUID: 733f0b2a-2dd6-45a4-80ec-bf2267f2a32d
Repo: https://gitlab.com/mosaik/api/MosaikAPIv3.jl.git
Tree: 7e09fbc71cf12d54c37821997d40a6070f715388

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

goerz commented:
[noblock] Thanks for updating the documentation and README. Both you and @rafaqz will have to unblock your comments by adding `[noblock]` in order for the merge to continue.
